pkg/oci/remote/remote.go

```diff
@@ -39,9 +40,17 @@ var (
 
 	// ErrEntityNotFound is the error that SignedEntity returns when the
 	// provided ref does not exist.
-	ErrEntityNotFound = errors.New("entity not found in registry")
+	ErrEntityNotFound = "cosign remoteGet: entity not found in registry"
```

Comment:
It's a little weird to have `ErrEntityNotFound` to just be a string. I also think you don't actually need this anymore if you make it a real error.